Snakefile_Regions

```diff
@@ -0,0 +1,184 @@
+include: "Snakefile"
+
+ruleorder: subsample_regions > subsample
+ruleorder: adjust_metadata_regions > adjust_metadata
+
+# How to run: if no region is specified, it'll run a subsampled global build (200 per division)
+# If a region is selected, it'll do 200/division for that region, and 20/division in the rest of the world
+#       -- preferentially sequences near the focal sequences
+#
+# To run a regional build, be sure to update it to the list in Snakefile
+#
+# You can run all builds in parallel!
+#   snakemake -s Snakefile_Regions --cores 2 all_regions
+#
+# Or you can specify final or intermediate output files like so:
+#   snakemake -s Snakefile_Regions --cores 2 auspice/ncov_europe.json (subsampled regional focal)
+#   snakemake -s Snakefile_Regions --cores 2 auspice/ncov.json (subsampled global)
+
+rule all_regions:
+    input:
+        auspice_json = expand("auspice/ncov{region}.json", region=REGIONS),
+        tip_frequencies_json = expand("auspice/ncov{region}_tip-frequencies.json", region=REGIONS),
+        dated_auspice_json = expand("auspice/ncov{region}_{date}.json", date=get_todays_date(), region=REGIONS),
+        dated_tip_frequencies_json = expand("auspice/ncov{region}_{date}_tip-frequencies.json", date=get_todays_date(), region=REGIONS),
+        auspice_json_gisaid = expand("auspice/ncov{region}_gisaid.json", region=REGIONS),
+        auspice_json_zh = expand("auspice/ncov{region}_zh.json", region=REGIONS)
+
+rule subsample_focus:
+    message:
+        """
+        Subsample all sequences into a focal set
+        """
+    input:
+        sequences = rules.mask.output.alignment,
+        metadata = rules.download.output.metadata,
+        include = files.include

```

Comment:
I think the `include`s should be part of the context filter, not the focal filter. If they are part of the focal group they have an unwanted effect on priority calculations if the focal group is far away from the root.